### DISCUSSION TITLE
Change cursor over disabled controls

### DISCUSSION THREAD

**Discussion by rabbitism** [AvaloniaUI/Avalonia#9640] [Q&A] [answered]
Is it possible to change cursor style for disabled control? 
It's not responding to any pointer movement, so setting cursor for `:pointerover` is not working.

**Comment by robloo** [comment, marked as answer]
This somewhat relates to https://github.com/AvaloniaUI/Avalonia/pull/9627 (@Takoooooo) and the issue/discussion in #7294.

The general consensus was disabled controls should be completely inert and uanble to receive and process events. You bring up a special case we may want to allow though. There are now two to consider:

 1. Some newer frameworks have `AllowDisabledFocus` property. This seems under debate but the reasoning is disabled controls may still **need to be focusable for accessibility** (think screen readers) -- although they still receive no input.
 2. Sometimes you may want to show a special cursor over disabled controls indicating they cannot be clicked or interacted with. (this discussion)

Since disabled controls don't receive any pointer events, I don't think it's currently possible to do what you are trying here.
